### DISCUSSION TITLE
can sea-orm-cli generate model support model field name  camel case option?

### DISCUSSION THREAD

**Discussion by ifnk** [SeaQL/sea-orm#2130] [Q&A]
here is my command 
`sea-orm-cli generate entity -o src/entity --with-serde both `
generate code like this 
![image](https://github.com/SeaQL/sea-orm/assets/39295458/de97c308-090b-456d-bf9b-66d5943493e5)
you can see it's field name is snake_case 
but i want name is camel case 
![image](https://github.com/SeaQL/sea-orm/assets/39295458/95f4b6f6-444b-49c4-b68b-48aa06eb5a32)

can sea-orm-cli generate model support model field name  camel case option ?

**Comment by tyt2y3** [comment]
No, snake case attribute names is a Rust convention.
